src/test/java/com/urlparser/entity/LinksFromUrlTest.java

```diff
@@ -0,0 +1,118 @@
+package com.urlparser.entity;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.urlparser.model.Links;
+
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.hamcrest.CoreMatchers;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class LinksFromUrlTest {
+
+  Links<URL> origin;
```

Comment:
@undsett this instance field is redundant, could you please move it to the test case?